shell/src/main/java/org/apache/accumulo/shell/commands/SetIterCommand.java

```diff
@@ -89,7 +88,7 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
     String tmpTable = null;
     String configuredName;
     try {
-      if (profileOpt != null && StringUtils.isBlank(shellState.getTableName())) {
+      if (profileOpt != null && shellState.getTableName().isBlank()) {
```

Comment:
I think this can be null.

Reply:
Do you know what this code is doing?  I don't understand why we are checking if its blank (null, empty or whitespace) and then assigning it if it is.  Then check again if its not empty after we just assign it...